simplq/src/components/pages/Admin/index.jsx

```diff
@@ -60,20 +61,38 @@ export default (props) => {
       .catch((err) => handleApiErrors(err));
   };
 
+  const HeaderSection = () => (
+    <div className={styles['header-bar']}>
+      <Header className={styles['header']} text={queueName} />
+      <div className={styles['main-button-group']}>
+        <div className={styles['admin-button']}>
+          <RefreshButton onClick={update} />
+        </div>
+        <div className={styles['admin-button']}>
+          <ShareQueue queueId={queueId} className={styles.shareButton} />
+        </div>
+      </div>
+    </div>
+  );
+
+  const Navbar = () => (
+    <div>
+      <nav className={styles['navbar']}>
+        <img src="public/LogoLight.png" alt="Home" />
+        {/* <p className={styles['simplq']}>SimplQ</p>
+        <p className={styles['sign-in']}>Sign In / Sign Up</p> */}
```

Comment:
We can let @arkumari2000  take care of this, if she is interested. 

My rough plan is to let her finish developing the navbar on the homepage, then she can refactor it and move it to common and reuse it.

After it's done, we can take a call as to if we should just reuse the css or if we can reuse the component by parameterising it.

Reply:
Sounds good.